libcloud.groovy

```diff
@@ -46,10 +48,89 @@ def replicate_to_clouds(pipecfg, basearch, buildID, stream) {
             """)
         }
     }
+    // A closure to build the aws-winli AMI. This will be called before replicating to 
+    // all regions, if the option is set for the current stream. `cosa aws-replicate` 
+    // will handle both traditional AMIs and aws-winli AMIs if present in the metadata. 
+    // aws-winli is only supported on x86_64.
+    def awsWinLIBuildClosure = { config, aws_image_name, credentialId ->
+        def creds = [file(variable: "AWS_CONFIG_FILE", credentialsId: credentialId)]
+        withCredentials(creds) {
+            utils.syncCredentialsIfInRemoteSession(["AWS_CONFIG_FILE"])
+            def c = config
+
+            // Since we are not uploading anything, let's just touch the vmdk image
+            // file to satisfy ore aws upload, which still requires the file in the
+            // cosa working dir.
```

Comment:
to be clear here. it's not actually `ore` it's the cosalib wrapper that checks to see if we "have the artifact" before attempting to upload.

Reply:
Thanks! I updated the comments.